Gruntfile.js

```diff
@@ -106,8 +106,10 @@ module.exports = function(grunt) {
         '!js/components.js',
         '!js/logging.js',
         '!js/backup.js',
+        '!js/i18n.js',
         '!js/modules/**/*.js',
         '!js/views/conversation_search_view.js',
+        '!js/react/**/*.js',
```

Comment:
Minor: Could we try and keep 🔡 order in such lists to make them easier to scan and avoid duplicates? I often use Sublime Text‘s <kbd>Sort Lines</kbd> command 😄